PresentScreenings/Entities/Screening.cs

```diff
@@ -305,15 +306,14 @@ public string ToScreeningLabelString(bool withDay = false)
 
         public string ToPlannedScreeningString()
         {
-            return string.Format("{0} {1}", ToLongTimeString(), ScreeningTitle);
+            return string.Format($"{ToLongTimeString()} {Screen} {ScreeningTitle}");
         }
 
         public string ToConsideredScreeningString()
         {
             string iAttend(bool b) => b ? "M" : string.Empty;
```

Comment:
Please replace magic letter with something like "the first letter of the film fan".

Reply:
Done.